tests/Unit/Evolution/Systems/RadiationTransport/M1Grey/Test_M1Closure.cpp

```diff
@@ -0,0 +1,98 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cmath>
+#include <cstddef>
+#include <utility>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/IndexType.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/RadiationTransport/M1Grey/M1Closure.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// Test tags used in M1Grey code
+SPECTRE_TEST_CASE("Evolution.Systems.RadiationTransport.M1Grey.M1Closure",
+                  "[Unit][M1Grey]") {
+  // Reference DataVector
+  const DataVector ref_dv(1);
```

Comment:
remove comment and rename to `used_for_size` and make it size 5

Reply:
Changed, but out of curiosity, why size 5 and not 1?

Reply:
Ok, thanks!